.compatibility_tests/compatibility_test_1_9/Cargo.toml

```diff
@@ -22,7 +22,7 @@ publish = false
 test_roqoqo_1_9 = { package = "roqoqo", version = "=1.9.0" }
 test_roqoqo_derive_1_9 = { package = "roqoqo-derive", version = "=1.9.0" }
 qoqo_calculator = { version = "~1.2" }
-roqoqo = { version = "~1.15", path = "../../roqoqo", features = [
+roqoqo = { version = "~1.15.2-alpha.0", path = "../../roqoqo", features = [
```

Comment:
Shouldn't we up the version of the compatibility_tests packages too?

Reply:
Good catch thanks!